Dockerfile

```diff
@@ -148,9 +150,11 @@ LABEL maintainer="Ilios Project Team <support@iliosproject.org>"
 
 # semi-colon seperates list of github users that can SSH in
 ENV GITHUB_ACCOUNT_SSH_USERS=''
+# set default LOCK_DSN for opensearch
+ENV LOCK_DSN="flock"
```

Comment:
This default should come out of the `.env` file. I'm worried we just need this one and not all the others in that file. Not sure why, but needs investigating. What was the path to getting a failure that made this needed?